### PR TITLE
Added AppImage build script for PyBitmessage

### PR DESCRIPTION
**Pull request by kdcis**

### CODE REVIEW

**Comment by g1itch** [comment]
Could you build the appimage on travis?

**Comment by g1itch** [comment]
Please don't include binary. It's build should be part of release process.

File: packages/AppImage/install.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+wget -O "appimagetool" "https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage"
+chmod a+x appimagetool
+mv appimagetool /usr/bin/
```
Comment (g1itch) [inline packages/AppImage/install.sh:5]: this requires root

Comment (kdcis) [reply packages/AppImage/install.sh:5]: Yes, is that a problem?. If yes I can change so it won't require root.

Comment (g1itch) [reply packages/AppImage/install.sh:5]: I'm not sure. Let's check: https://github.com/g1itch/PyBitmessage/commits/ci-wine-further

**Comment by g1itch** [comment]
I see some issue with script: https://travis-ci.org/github/g1itch/PyBitmessage/jobs/773056006, need to check it manually as well.

Also mark the last commit as closing #1450 when finish.

**Comment by g1itch** [comment]
OK, I found possible fix for AppRun: efefa4f. Bad news: this appimage still requires PyQt4 installed system wide.

File: packages/AppImage/install.sh (hunk 1 above)
Comment (g1itch) [reply packages/AppImage/install.sh:5]: It should work without root: a67ad09

Comment (kdcis) [reply packages/AppImage/install.sh:5]: OKay, I'll make the changes.

**Comment by g1itch** [comment]
Please squash two first commits to not bloat the repo. Also we need dependencies inside of the bundle, because its purpose is to work around the removal of PyQt4 and python2.

**Pull request closed without merging**